test/e2e/workflow/validation_test.go

```diff
@@ -13,6 +13,9 @@ import (
 
 var _ = Describe("NetworkAddonsConfig", func() {
 	gvk := GetCnaoV1GroupVersionKind()
+	AfterEach(func() {
```

Comment:
Since the Contexts that create it are the inner contexts, 
wouldn't it be better to add it for them ?
it will repeat twice, but the scope will be more accurate